CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Breaking Changes
+- `handler-slack-multichannel.rb`: Fixed title unknown issue, change from client address to client name
+
+## [Unreleased]
```

Comment:
I will clean up the duplicate `[Unreleased]` when I have a minute and then will merge and release.